core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/Stack.java

```diff
@@ -251,6 +252,10 @@ public class Stack implements ProvisionEntity, WorkspaceAwareResource, Orchestra
      */
     private String originalName;
 
+    @OneToOne(cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "database_id")
+    private Database database;
+
```

Comment:
I think currently we have databaseCrn in Cluster entity, and also in SdxCluster in datalake service. wouldn't this be to complex to understand?

Reply:
To move the dbcrn from cluster to the new database property would be too much effort, so we decided not to do it, I will raise a followup ticket for this